lib/MwbExporter/Formatter/Doctrine2/Annotation/Model/Column.php

```diff
@@ -177,7 +182,9 @@ public function asAnnotation()
         if ($this->isUnsigned()) {
             $attributes['options'] = array('unsigned' => true);
         }
-
+        if ($this->isUnique) {
```

Comment:
Why have you had to add the unique attribute in here? This seems disconnected with the DC2Type incorporation... Is there a separate bug?

Reply:
@peterchallett Quite right about the disconnect, this is hangover code from a different experiment and shouldn't be here.

When annotating an entity property, one way to specify that the column should be unique is to set the `unique` specifier to true, e.g.

`@ORM\Column(type="guid", unique=true)`

I noticed that the doctrine exporter doesn't do this, even when the `unique` option is checked for a given field within MySQL workbench. This is fine, however, because the doctrine exporter sets a table-level unique constraint instead, e.g.

`@ORM\Table(name="<table name>", uniqueConstraints={@ORM\UniqueConstraint(name="<col>_UNIQUE", columns={"<col>"})})`